kubernetes-discovery-client/src/main/java/io/micronaut/kubernetes/configuration/KubernetesConfigurationClient.java

```diff
@@ -66,19 +72,21 @@
     public static final String KUBERNETES_CONFIG_MAP_LIST_NAME = "Kubernetes ConfigMapList";
     public static final String KUBERNETES_CONFIG_MAP_NAME_SUFFIX = " (Kubernetes ConfigMap)";
     public static final String KUBERNETES_SECRET_NAME_SUFFIX = " (Kubernetes Secret)";
+    public static final String OPAQUE_SECRET_TYPE = "Opaque";
 
     private static final Logger LOG = LoggerFactory.getLogger(KubernetesConfigurationClient.class);
 
     private static Map<String, PropertySource> propertySources = new ConcurrentHashMap<>();
 
-    private final KubernetesClient client;
+    private final CoreV1ApiReactorClient client;
     private final KubernetesConfiguration configuration;
 
     /**
-     * @param client        An HTTP Client to query the Kubernetes API.
+     * @param client        An Core HTTP Client to query the Kubernetes API.
      * @param configuration The configuration properties
      */
-    public KubernetesConfigurationClient(KubernetesClient client, KubernetesConfiguration configuration) {
+    public KubernetesConfigurationClient(CoreV1ApiReactorClient client, KubernetesConfiguration configuration) {
+        LOG.info("MRDKAAAAAAAAAAAAAA");
```

Comment:
@pgressa oops?

Reply:
@rdesgroppes thanks for reporting. I have removed the snippet: 

https://github.com/micronaut-projects/micronaut-kubernetes/commit/2bbd37906eef70f838d3a653ba098492e65b870b

Reply:
@rdesgroppes thanks!